group_vars/all.yml.sample

```diff
@@ -726,6 +726,7 @@ dummy:
 #dashboard_rgw_api_user_id: ceph-dashboard
 #dashboard_rgw_api_admin_resource: ''
 #dashboard_rgw_api_no_ssl_verify: False
+#dashboard_frontend_vip: ''
```

Comment:
I think you added this manually but you shouldn't have to do that (directly in the sample files)
The right way to add new default value is:
- add the variable in `roles/ceph-defaults/defaults/main.yml`
- run the `generate_group_vars_sample.sh` script (that will generated the all.yml.sample and rhcs.yml.sample)

Reply:
Sorry @dsavineau, I miss the process of variable creation through the `generate_group_vars_sample.sh` script, /me updated the PR, can you double check I don't miss other things?